crates/rustc_codegen_spirv/src/symbols.rs

```diff
@@ -129,21 +129,21 @@ const BUILTINS: &[(&str, BuiltIn)] = {
         ("bary_coord_no_persp_nv", BaryCoordNoPerspNV),
         ("frag_size_ext", FragSizeEXT),
         ("frag_invocation_count_ext", FragInvocationCountEXT),
-        ("launch_id_nv", LaunchIdNV),
-        ("launch_size_nv", LaunchSizeNV),
-        ("world_ray_origin_nv", WorldRayOriginNV),
-        ("world_ray_direction_nv", WorldRayDirectionNV),
-        ("object_ray_origin_nv", ObjectRayOriginNV),
-        ("object_ray_direction_nv", ObjectRayDirectionNV),
-        ("ray_tmin_nv", RayTminNV),
-        ("ray_tmax_nv", RayTmaxNV),
-        ("instance_custom_index_nv", InstanceCustomIndexNV),
-        ("object_to_world_nv", ObjectToWorldNV),
-        ("world_to_object_nv", WorldToObjectNV),
-        ("hit_t_nv", HitTNV),
-        ("hit_kind_nv", HitKindNV),
-        ("incoming_ray_flags_nv", IncomingRayFlagsNV),
+        ("launch_id_khr", LaunchIdNV),
+        ("launch_size_khr", LaunchSizeNV),
+        ("instance_custom_index_khr", InstanceCustomIndexNV),
```

Comment:
khr on left side, NV on right? were some NV names reused in the KHR extension?

Reply:
It's just because rspirv has the NV versions but not the KHR versions.

Reply:
Probably best to wait for an rspirv update then. The NV things are _probably_ the same as the KHR ones but it's best no to mix them up. Looking foward to this though!

Reply:
Huh? rspirv has the KHR versions already though https://docs.rs/spirv_headers/1.5.0/spirv_headers/enum.BuiltIn.html#associatedconstant.LaunchIdKHR

Reply:
Ah, I figured it out. They're not variants but constants. So you can't use `use Builtin::*;` and then say `LaunchIdKHR` because it's not imported with the variants. I've updated the PR to use those constants.